spec/ameba/rule/style/verbose_block_spec.cr

```diff
@@ -165,9 +167,6 @@ module Ameba::Rule::Style
         %|map(&.to_s.[start: 0.to_i64, count: 3]?)|,
         %|map(&.to_s.[0.to_i64, count: 3]?)|,
         %|map(&.to_s.[0.to_i64, 3]?)|,
-        %|map(&.to_s.[start: 0.to_i64, count: 3]=("foo"))|,
```

Comment:
It feels weird to just rip out test cases, but if we introduce a new default that would make certain scenarios non-issues, is this the right approach?

Reply:
Nope, all of these should stay as they're testing the formatting, regardless of the default setting.

Reply:
The trouble is that I think this uses the defaults for running the checks, so these three lines weren't showing as issues when I changed the default value.

Should I move these test cases up to `#exclude_operators`? It was my assumption that _that_ was where we'd put formatting tests that might change based on the settings.

Reply:
No, these belong here as they test the formatting, not the option behavior. Since the default changed, just set it back to needed value at the beginning of the test.